src/terraform/commands.go

```diff
@@ -10,6 +10,9 @@ import (
 
 // RunInit runs the 'terraform init' command and force copies the remote state
 func RunInit(deployConfig types.DeployConfig) error {
+	if !util.DoesFileExist(deployConfig.TerraformFile) {
+		return errors.New("terraform initialization attempted in directory with no 'main.tf' file. Please make sure 'exo deploy build' has been run")
+	}
```

Comment:
I meant if you run `exo deploy apply` without running `exo deploy build` then it runs `exo deploy build` for you instead of telling you to run it. If the `main.tf` exists, it just applies it.